jenkins-pipelines/oss/longevity/longevity-100gb-2h-rackaware.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    availability_zone: 'a,c',
```

Comment:
I think this is wrong: we should use 3 racks to have RF=RACKS

Reply:
Why do we need RF=RACKS? Is it must configuration?

Reply:
it's some kind of current scylla limitation I see here and there (but actually, i don't recall any explanation why we have this restriction). Search email for `RF=#racks`.

Reply:
@soyacz do you mean this?

```
With Tablets, the Replication Factor (RF) cannot be updated to a value higher than 
the number of nodes per Data Center (DC).
```

https://docs.google.com/document/d/1LRWkg3oxLcC3n_GolbqBLtEO_jsvV6JObewlc9PIi4M/edit?tab=t.0

Reply:
Looks like I found the email: subject "2025.1.0 blocker - cql3: Introduce RF-rack restricted keyspaces "

```
With the upcoming (2025.1.0) changes to tablet migration, we would like to enforce that RF# == #racks (incl. support of virtual racks in Scylla cloud, where even a single physical zone has 3 racks).

Per DC RF is equal to the number of racks in that DC.
```

Reply:
@fruch 
According to above we need to change multiDC test to run with 3 racks. It means we need to increase the nodes amount as well. Do you agree?

Reply:
it mean that we should have 3 or more nodes in each DC, and we need work with `a,b,c` or with `simulated_racks: 3`

Reply:
change to 3 racks in every DC with 2 nodes in every RAC